app/views/people/person_tasks.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= "#{@person.name}'s Tasks" %></h1>
+
+<table>
+  <tr>
+    <th> Name </th>
+    <th> Description </th>
+    <th> Completed? </th>
+  </tr>
+  <% @tasks.each do |task| %>
+    <tr>
+      <td><%= task[:name] %></td>
```

Comment:
Keep in mind consistency when you retrieve your model fields. Preference should go to `model.field` over `model[:field]`